tests/Functional/ConnectionTest.php

```diff
@@ -291,10 +291,12 @@ public function testTransactional(): void
     {
         $this->createTestTable();
 
+        /** @psalm-suppress AssignmentToVoid */
         $res = $this->connection->transactional(static function (Connection $connection): void {
             $connection->insert(self::TABLE, ['id' => 2]);
         });
 
+        /** @psalm-suppress RedundantCondition */
         self::assertNull($res);
```

Comment:
I believe, it will be:
```suggestion
        self::assertEquals(1, $res)
```